apis/core/lib/domain/cube-projects/create.ts

```diff
@@ -13,21 +12,17 @@ interface CreateProjectCommand {
 export async function createProject({ resource, store }: CreateProjectCommand): Promise<GraphPointer> {
   const label = resource.out(rdfs.label).term!
 
-  const project = clownface({ dataset: $rdf.dataset() })
-    .namedNode(id.cubeProject(label.value))
+  const project = store.create(cubeProject(label.value))
```

Comment:
Small change but I think this produces a much more readable effect

Reply:
Nice!